packages/components/src/components/text-field/text-field.tsx

```diff
@@ -160,6 +171,7 @@ export class TextField {
             minLength={this.minLength}
             maxLength={this.maxLength}
             id={this.inputId}
+            {...list}
```

Comment:
This can be `list={this.list}`. If `this.list` is undefined, the attribute won't render.

Reply:
@acstll done.